docassemble/PDFStats/data/static/upload_file.html

```diff
@@ -25,10 +25,17 @@
   Suffolk LIT Lab: Rate My PDF
   </a>
   </div>
-</nav>    
+</nav>
 <main class="upload-centered">
+  <h1 class="h3 mb-3 fw-normal">Rate your PDF form</h1>
+    <p>
+      Select a PDF (with form fields) to upload, and RateMyPDF will give you a
+      score and detailed recommendations to make your form simpler and easier
+      for a litigant to complete. <i>Note: RateMyPDF doesn't work with Adobe LiveCycle
```

Comment:
```suggestion
      for a person to complete. <i>Note: RateMyPDF doesn't work with Adobe LiveCycle
```
I think we should either be super general, and not mention legal specific things (which there aren't many legal-specific metrics, just citations), or go in the other direction and say that this is specifically for legal forms.